.pylintrc

```diff
@@ -75,3 +75,9 @@ disable=
 	# NOTE(nathaniel): I have disputed the premise of this inspection from
 	# the beginning and will continue to do so until it goes away for good.
 	useless-else-on-loop,
+	# NOTE(mmx): Our existing code is written to favor explicit else over
+	# relying on return within a branch.
+	no-else-return,
+	# NOTE(mmx): Disable unnecessary super-init requirement for abstract
```

Comment:
This is probably worth being a `TODO(https://github.com/PyCQA/pylint/issues/<integer>): Enable this inspection.`.

Reply:
Perhaps, but this sounded like having too many false positives. I'll turn it to a TODO though. I agree we should be adding superfluous `super.__init__` even if we just do so to be verifiably correct.